style/ruby/README.md

```diff
@@ -27,6 +27,7 @@ Ruby
 * Prefer `if` over `unless`.
 * Use `_` for unused block parameters.
 * Prefix unused variables or parameters with underscore (`_`).
+* Use a leading underscore when defining instance variables for memoization.
```

Comment:
From the README:

> "Prefer" indicates a better option and its alternative to watch out for.
> "Use" is a positive instruction.

Use makes more sense to me, as the alternative is just "don't use an underscore."